projects/storefrontlib/src/cms-components/asm/asm-root/asm-root.component.ts

```diff
@@ -35,12 +15,4 @@ export class AsmRootComponent implements OnInit, OnDestroy {
   collapseUi(): void {
     this.asmService.updateAsmUiState({ expanded: false });
   }
```

Comment:
With the visibility now handled in AsmMainUiComponent, we would likely need to handle collapse/expand in AsmMainUiComponent because when Asm is disabled or closed, we need the expand buttons to disappear as well.

Either that or we can emit an event from AsmMainUiComponent to AsmRootComponent so the AsmRootComponent handles visibility.

Depending on what we chose, the AsmRootComponent might end up essentially empty and we'll need to decide if we keep it around at all.

Reply:
Agreed, I've dropped the root component. I did had to refactor the enabler service a bit to avoid cyclic deps.